scripts/update/Updater.php

```diff
@@ -350,5 +350,27 @@ public function update($initialVersion) {
         }
 
         $this->skip('8.0.0', '8.1.0');
+
+        if ($this->isVersion('8.1.0')) {
+            /** @var \common_persistence_Manager $persistenceManager */
+            $persistenceManager = $this->getServiceManager()->get(\common_persistence_Manager::SERVICE_ID);
+
+            $persistenceManagerConfig = $persistenceManager->getOption('persistences');
+
+            $pdoDriverPrefix = 'pdo_';
+            foreach ($persistenceManagerConfig as $persistenceId => $persistenceParams) {
+                if (substr($persistenceParams['driver'], 0, strlen($pdoDriverPrefix)) === $pdoDriverPrefix) {
```

Comment:
`strpos($persistenceParams['driver'], $pdoDriverPrefix) === 0` a bit shorter and readable :)